bootstrap.sample.json

```diff
@@ -25,5 +25,44 @@
       "_id": "local",
       "type": "engine"
     }
+  ],
+  "filetypes": [
+    { "_id": "BVAL",            "regex": "\\.(bval|bvals)$" },
+    { "_id": "BVEC",            "regex": "\\.(bvec|bvecs)$" },
+    { "_id": "DICOM",           "regex": "\\.(dcm|dcm\\.zip|dicom\\.zip)$" },
+    { "_id": "EFile",           "regex": "^E.*P.*\\.7$" },
+    { "_id": "GE Physio",       "regex": "\\.gephysio\\.zip$" },
+    { "_id": "MGH Data",        "regex": "\\.(mgh|mgz|mgh\\.gz)$" },
+    { "_id": "NIfTI",           "regex": "\\.(nii\\.gz|nii)$" },
+    { "_id": "PAR/REC",         "regex": "\\.(parrec\\.zip|par-rec\\.zip)$" },
+    { "_id": "PFile Header",    "regex": "\\.(7\\.hdr)$" },
+    { "_id": "PFile",           "regex": "\\.(7\\.gz|7|7\\.zip)$" },
+
+    { "_id": "EEG",             "regex": "\\.eeg\\.zip$" },
+
+    { "_id": "QC",              "regex": "\\.(q[ac]\\.png|q[ac]\\.json|q[ac]\\.html)$" },
+
+    { "_id": "MATLAB Data",     "regex": "\\.mat$" },
+    { "_id": "PsychoPy Data",   "regex": "\\.psydat$" },
+
+    { "_id": "Archive",         "regex": "\\.(zip|tbz2|tar\\.gz|tbz|tar\\.bz2|tgz|tar|txz|tar\\.xz)$" },
+    { "_id": "Audio",           "regex": "\\.(mp3|wav|wave)$" },
+    { "_id": "Document",        "regex": "\\.(docx|doc)$" },
+    { "_id": "HDF5",            "regex": "\\.(h5|hdf5)$" },
+    { "_id": "Image",           "regex": "\\.(jpg|tif|jpeg|gif|bmp|png|tiff)$" },
+    { "_id": "JSON",            "regex": "\\.json$" },
+    { "_id": "Log",             "regex": "\\.log$" },
+    { "_id": "Markdown",        "regex": "\\.(md|markdown)$" },
+    { "_id": "Markup",          "regex": "\\.(html|htm)$" },
+    { "_id": "PDF",             "regex": "\\.pdf$" },
+    { "_id": "Plain Text",      "regex": "\\.txt$" },
+    { "_id": "Presentation",    "regex": "\\.(ppt|pptx)$" },
+    { "_id": "Source code",     "regex": "\\.(c|py|cpp|js|m|json|java|php|css|toml|yaml|yml)$" },
+    { "_id": "Spreadsheet",     "regex": "\\.(xls|xlsx)$" },
+    { "_id": "TOML",            "regex": "\\.toml$" },
+    { "_id": "Tabular data",    "regex": "\\.([ct]sv\\.gz|[ct]sv)$" },
+    { "_id": "Video",           "regex": "\\.(mpeg|mpg|mov|mp4|m4v|mts)$" },
+    { "_id": "XML",             "regex": "\\.xml$" },
+    { "_id": "YAML",            "regex": "\\.(yaml|yml)$" }
```

Comment:
Megan alerted me to this change: I don't think we can do this since it would break the gear surface area. Suggest using the original names in the `filetypes.json` file

Reply:
@kofalt Thanks for the heads up. Could you please elaborate? What are all the aspects you see impacted by this change?

Anything beyond this?
- massive DB migration
- manifest updates
- all gears updated at all sites

Reply:
So, my understanding because these filetypes are presented to gears, and they might rely on that field, a check for `== "MATLAB data"` would fail if the string is now `"MATLAB Data"`.

This is a good example of how we don't quite have the full gear "surface area" mapped out and committed to: with our quickly-built deep hierarchy integration, there's more that could make a gear fail through no fault of its own.

So our options (I think) are:

1. Not change existing strings
1. Change the strings, but have the old versions for when they're provided externally (?)
1. Triage the full set of gears & gear authors, communicate a breaking change